lib/OpenQA/WebAPI/Controller/LiveViewHandler.pm

```diff
@@ -37,8 +38,11 @@ my %allowed_os_autoinst_commands = (
 # define global variable to store ws connections (tx) to isotovideo cmd srv for each job
 my %cmd_srv_transactions_by_job;
 
-# define global variable to store ws connections (tx) to JavaScript clients for each job
-my %java_script_transactions_by_job;
+# define global variable to store development session ws connections (tx) to JavaScript clients for each job
+my %devel_java_script_transactions_by_job;
```

Comment:
This can be easily converted to a property of the controller via has: `has blah => sub { {} };`

Reply:
So if you want to refactor this, the property must be on the application. But for now I would say there are more important things to adjust.